File: problema05/sol_python/Problema05.py
x = float(input("Ingrese el valor de X: "))
y = float(input("Ingrese el valor de Y: "))
if x >= 0 and y >= 0:
	print("Valores pertenecientes al cuadrante 1.")
elif x < 0 and y >= 0:
	print("Valores pertenecientes al cuadrante 2.")
elif x < 0 and y < 0:
	print("Valores pertenecientes al cuadrante 3.")
else:
	print("Valores pertenecientes al cuadrante 4.")			

File: problema05/sol_java/Problema05/src/problema05/Problema05.java

package problema05;

import java.util.Scanner;
import java.util.Locale;

public class Problema05 {
    public static void main(String[] args) {
        
        Scanner entrada = new Scanner(System.in);
        entrada.useLocale(Locale.US);
        
        double x;
        double y;
        
        System.out.println("Ingrese el valor de X: ");
        x = entrada.nextDouble();
        System.out.println("Ingrese el valor de Y: ");
        y = entrada.nextDouble();
        
        if (x >= 0 && y >= 0){
            System.out.print("Valores pertenecientes al cuadrante 1.\n");
        }else if(x < 0 && y >= 0){
            System.out.print("Valores pertenecientes al cuadrante 2.\n");
        }else if(x < 0 && y < 0){
            System.out.print("Valores pertenecientes al cuadrante 3.\n");
        }else{
            System.out.print("Valores pertenecientes al cuadrante 4.\n");
        }
    }
    
}
